lexicons/app/bsky/authDeletePosts.json

```diff
@@ -0,0 +1,25 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.authDeletePosts",
```

Comment:
if we're taking one or the other, i lean towards `authManagePosts` than `authDeletePosts`